### ISSUE TITLE
Livepeer Command Line Tool

### ISSUE THREAD

**Issue by ericxtang** [livepeer/livepeer-swarm#46]
There should be:

`livepeer daemon` - launches livepeerd
`livepeer stream -d {deviceID} -h {hostInfo}` - tries to capture a video stream from a local device and streams to host
`livepeer view -s {streamID}` - tries to view a stream (requires a running local daemon)

**Comment by dob** [comment]
Are these designs for the next iteration of the node? We already have 

`livepeer` which launches the node
`livepeer stream <id>`which does what you describe as `view` above. 

See `cmd/livepeer/main.go`

**Comment by ericxtang** [comment]
Right.  I think people get confused about the OBS stuff - probably much easier if we just issued the FFMPEG command to start the view stream and hide all the complexity (especially for beginners).

Couldn't think of a good command name for "start the stream" besides `stream` (although it's a little confusing - could either mean "start the stream" or "view the stream").  Or maybe we should use `broadcast` and `view`.

Also, I think we should add commands for diagnostics, checking dependencies, etc.
